src/appmixer/bigCommerce/artifacts/ai-artifacts/README.md

```
@@ -0,0 +1,172 @@
+# BigCommerce Connector
+
+## Generated Artifacts
+
+**Service Info Actions and Triggers:**
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)